pkg/config/config_test.go

```diff
@@ -41,3 +45,9 @@ func TestConfig(t *testing.T) {
 		t.Fatalf("GetEbpfEngineLoaderPath failed")
 	}
 }
+
+func currentDir() string {
+	_, filename, _, _ := runtime.Caller(1)
```

Comment:
https://pkg.go.dev/os#Getwd ?

Reply:
I have copy it from the kubescape. I prefer to use in the same method that other kubescape projects use